### PR TITLE
fix argument parsing for --scenarios

### PR DESCRIPTION
**Pull request by fliiiix**

### CODE REVIEW

File: radish/main.py [hunk 1]
```diff
@@ -76,7 +76,9 @@ def run_features(core):
     # scenario choice
     amount_of_scenarios = sum(len(f.scenarios) for f in core.features_to_run)
     if world.config.scenarios:
-        world.config.scenarios = [int(s) for s in world.config.scenarios.split(",")]
+        world.config.scenarios = [
+            int(s.strip().replace("=", "")) for s in world.config.scenarios.split(",")
```
Comment (timofurrer) [inline radish/main.py:80]: Does this also happen for docopt?

Comment (fliiiix) [reply radish/main.py:80]: If you mean that it breaks if you use `-s=1` the answer is yes (if not then I don't get your question)

**Comment by fliiiix** [comment]
Can I merge and release this tomorrow?

**Comment by timofurrer** [comment]
Yes, go for it :)

**Pull request merged**